content/en/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -0,0 +1,369 @@
+---
+title: API Priority and Fairness
+content_template: templates/concept
+min-kubernetes-server-version: v1.18
+---
+
+{{% capture overview %}}
+
+Controlling the behavior of the Kubernetes API server in an overload situation
+is a key task for cluster administrators. The {{< glossary_tooltip
+term_id="kube-apiserver" text="kube-apiserver" >}} has some controls available
+(i.e. the `--max-requests-inflight` and `--max-mutating-requests-inflight`
+command-line flags) to limit the amount of outstanding work that will be
+accepted, preventing a flood of inbound requests from overloading and
+potentially crashing the API server, but these flags are not enough to ensure
+that the most important requests get through in a period of high traffic.
+
+The API Priority and Fairness feature is an alternative to the
+aforementioned max-inflight limitations.  This new feature classifies
+and isolates requests in a more fine-grained way.  It also introduces
+a limited amount of queuing, so that no requests are rejected in cases
+of very brief bursts.  Requests are dispatched from queues using a
+fair queuing technique so that, for example, a poorly-behaved {{<
+glossary_tooltip text="controller" term_id="controller" >}}) need not
+starve others (even at the same priority level).
+
+{{< caution >}}
+In the present implementation, requests classified as "long-running" — primarily
+watches — are not subject to the API Priority and Fairness filter. This is also
+true for the `--max-requests-inflight` flag without the API Priority and
+Fairness feature enabled.
+{{< /caution >}}
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Enabling API Priority and Fairness
+
+{{< feature-state state="alpha"  for_k8s_version="v1.18" >}}
```

Comment:
I recommend moving this above the heading “Enabling API Priority and Fairness” so that it's the first thing in the page body. Bear in mind that the feature-state shortcode will remain even after “Enabling API Priority and Fairness” is removed.

Reply:
done